src/js/components/Form.js

```diff
@@ -22,7 +29,7 @@ class Form extends Component {
 
   isValid = () => {
     const { changeset } = this.props
-    return ["assignment_id", "task_id", "hours"].map(prop => changeset[prop]).every(Boolean)
+    return ["assignment_id", "task_id"].map(prop => changeset[prop]).every(Boolean)
```

Comment:
hours are now optional, the submit button will now also be active when the hours input field is empty.